app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -88,7 +88,10 @@ class MS.Views.TimelineTableView extends Backbone.View
 
   addDependentStep: (e) ->
     e.preventDefault()
-    @stepModal.new(@projectId, @refresh.bind(@), precedentId: @stepIdFromEvent(e))
+    @stepModal.new(@projectId, @refresh.bind(@),
+      precedentId: @stepIdFromEvent(e),
+      parentId: @stepParentIdFromEvent(e)
+    )
```

Comment:
Note the small stylistic change. It's more common to split just the hash arguments across lines while leaving the regular ones inline, unless the regular ones are super long too.